angular-app/src/app/workspace/workspace.component.html

```diff
@@ -1,21 +1,10 @@
 <mat-sidenav-container class="component-drawer-container">
   <mat-sidenav-content>
-    <div id="new-workspace-prompt">
-        <h1 id="prompt">Create a New Project?</h1>
-        <!-- TODO #47: Link up New Project Button with Gwyneth's function -->
-        <button id="new-project-button" (click)="newProject()">New Project</button>
-    </div>
-    <div id="workspace-content" class="hidden">
-        <button class="save-button">Save</button>
-        <div id="project-name" class="center">
-            <mat-form-field class="center input-project-name">
-                    <mat-label>Project Name</mat-label>
-                    <textarea matInput
-                        cdkTextareaAutosize
-                        #autosize="cdkTextareaAutosize"
-                        cdkAutosizeMinRows="1"
-                        cdkAutosizeMaxRows="2"></textarea>
-                </mat-form-field>
+    <div id="workspace-content">
+        <button class="right-button" (click)="newProject()">Create New Project</button>
```

Comment:
You could change your buttons to mat-buttons for them to be formatted more consistently with the app.

https://material.angular.io/components/button/overview